lib/CHeroHandler.h

```diff
@@ -340,8 +310,6 @@ class DLL_LINKAGE CHeroHandler : public CHandlerBase<HeroTypeID, HeroType, CHero
 		h & expPerLevel;
 		h & ballistics;
 		h & terrCosts;
-		h & obstacles;

```

Comment:
Need to check if they are restored correctly after game load. For instance if save has less obstacles than current configs

Reply:
But it's just serialised in another place (in ObstacleHandler)